tweets/views.py

```diff
@@ -16,17 +16,29 @@ def showTweet(request,id):
 
 def editTweet(request,id):
   tweet = get_object_or_404(Tweet, id=id, user_id=request.user.id)
+  if tweet.original_tweet is not None:
+   return HttpResponse("Unprocessable Tweet", status=422)
   return render(request, 'tweets/edit.html',{'tweet':tweet})
 
 def updateTweet(request,id):
   tweet = get_object_or_404(Tweet, id=id, user_id=request.user.id)
+  if tweet.original_tweet is not None:
+   return HttpResponse("Unprocessable Tweet", status=422)
```

Comment:
Correct status, wrong message

Reply:
Unable to retweet a tweet that has already been retweeted.

Reply:
how about this

Reply:
> Unable to retweet a tweet that has already been retweeted.
WHY Unable ?

Unable means that the application is not able to handle this kind of request. And User will have no idea why the application is `Unable to retweet a tweet that has already been retweeted.`

How about you try something better  ?

Reply:
``` Retweeted tweets can't be retweeted ``` 
``` You can't tweet a tweet which is already retweeted ``` 
``` You can't tweet a retweet ``` 

Although this error will only show if a user try to hit urls manually by typing because the retweet link automatically hide on tweets which are retweeted.

Reply:
>Retweeted tweets can't be retweeted
LGTM